llvm/test/CodeGen/AMDGPU/load-constant-always-uniform.ll

```diff
@@ -0,0 +1,42 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1100 < %s | FileCheck %s -prefix=GFX11
```

Comment:
Perhaps this should also have `gfx1200` run line to show that selection on GFX12 uses `s_load_b96`?